### PR TITLE
PSMDB-1257 Release notes 6.0.6-5

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   .github/workflows/main.yml
	new file:   docs/release_notes/6.0.6-5.md
	modified:   docs/release_notes/index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release_notes/6.0.6-5.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+| Release date | May , 2023|
+|------------- | ---------------|
+| **Installation** | [Installing Percona Server for MongoDB](../install/index.md) |
+
+
+Percona Server for MongoDB 6.0.6-5 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 6.0.6 Community Edition.
+
+It is rebased on [MongoDB 6.0.6 Community edition](https://www.mongodb.com/docs/manual/release-notes/6.0/#6.0.6---mar-13--2023) and supports the upstream protocols and drivers.
```
Comment (igorsol) [inline docs/release_notes/6.0.6-5.md]: I don't think "rebased" is correct word for PSMDB. Can we return to "based" wording?

**Review by igorsol — CHANGES_REQUESTED** [review]

File: docs/release_notes/6.0.6-5.md (hunk 1 above)
Comment (nastena1606) [reply docs/release_notes/6.0.6-5.md]: Done

File: docs/release_notes/6.0.6-5.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Find the full list of new features and improvements in [MongoDB 6.0.6 Community edition release notes](https://www.mongodb.com/docs/manual/release-notes/6.0/#6.0.6---).
+
+
+## Bugs Fixed
+
+## Bugs Fixed
+
+* [PSMDB-1211](https://jira.percona.com/browse/PSMDB-1211): Improved the master key rotation handling in case of failure
+* [PSMDB-1231](https://jira.percona.com/browse/PSMDB-1231): Register a master key for data-at-rest encryption encryption on the KMIP server in the raw-bytes form
```
Comment (igorsol) [inline docs/release_notes/6.0.6-5.md]: "encryption" is doubled

**Review by igorsol — CHANGES_REQUESTED** [review]

File: docs/release_notes/6.0.6-5.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Percona Server for MongoDB 6.0.6-5 (2023-05-18)
+
+| Release date | May 18, 2023|
```
Comment (sandraromanchenko) [inline docs/release_notes/6.0.6-5.md]:
```suggestion
| Release date | May 25, 2023|
```

File: mkdocs-base.yml [hunk 4]
```diff
@@ -101,7 +101,7 @@ plugins:
   with-pdf: # https://github.com/orzih/mkdocs-with-pdf
           output_path: '_pdf/PerconaServerforMongoDB-6.0.pdf'
           cover_title: 'Server for MongoDB Documentation 6.0'
-          cover_subtitle: 6.0.5-4 (March 29, 2023)
+          cover_subtitle: 6.0.6-5 (May 18, 2023)
```
Comment (sandraromanchenko) [inline mkdocs-base.yml]:
```suggestion
          cover_subtitle: 6.0.6-5 (May 25, 2023)
```

File: variables.yml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # PBM Variables set for HTML output
 # See also mkdocs.yml plugins.with-pdf.cover_subtitle and output_path
 
-release: '6.0.5-4'
+release: '6.0.6-5'
 version: '6.0'
-release_date: 2023-03-29
+release_date: 2023-05-18
```
Comment (sandraromanchenko) [inline variables.yml]:
```suggestion
release_date: 2023-05-25
```

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

File: docs/release_notes/6.0.6-5.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+| Release date | May 25, 2023|
+|------------- | ---------------|
+| **Installation** | [Installing Percona Server for MongoDB](../install/index.md) |
+
+
+Percona Server for MongoDB 6.0.6-5 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 6.0.6 Community Edition.
+
+It is based on [MongoDB 6.0.6 Community edition](https://www.mongodb.com/docs/manual/release-notes/6.0/#6.0.6---mar-13--2023) and supports the upstream protocols and drivers.
```
Comment (sandraromanchenko) [inline docs/release_notes/6.0.6-5.md]:
```suggestion
It is based on [MongoDB 6.0.6 Community edition](https://www.mongodb.com/docs/manual/release-notes/6.0/#6.0.6---may-12--2023) and supports the upstream protocols and drivers.
```

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

**Review by sandraromanchenko — APPROVED** [review]

**Review by igorsol — APPROVED** [review]

**Review by sandraromanchenko — APPROVED** [review]

**Pull request merged**